tests/storage/cdi_upload/test_upload_virtctl.py

```diff
@@ -252,34 +252,28 @@ def test_virtctl_image_upload_pvc(download_image, namespace, storage_class_name_
 
 
 @pytest.mark.sno
-@pytest.mark.polarion("CNV-3725")
-def test_virtctl_image_upload_with_exist_dv(
-    unprivileged_client, download_image, namespace, storage_class_name_scope_module
-):
-    """
-    Check that virtctl is able to upload a local disk image to an existing DataVolume
-    """
-    dv_name = "cnv-3725"
-    with create_dv(
-        source="upload",
-        dv_name=dv_name,
-        namespace=namespace.name,
-        size=DEFAULT_DV_SIZE,
-        storage_class=storage_class_name_scope_module,
-    ) as dv:
-        dv.wait_for_status(status=DataVolume.Status.UPLOAD_READY, timeout=120)
-        with virtctl_upload_dv(
-            namespace=namespace.name,
-            name=dv.name,
-            size=DEFAULT_DV_SIZE,
-            image_path=LOCAL_PATH,
-            insecure=True,
-            storage_class=storage_class_name_scope_module,
-            no_create=True,
-        ) as res:
-            check_upload_virtctl_result(result=res)
-            if not sc_volume_binding_mode_is_wffc(sc=storage_class_name_scope_module):
-                create_vm_from_dv(client=unprivileged_client, dv=dv, start=True)
+class TestVirtctlUploadExistingDV:
+    @pytest.mark.polarion("CNV-3725")
+    def test_virtctl_image_upload_to_existing_dv_and_create_vm(
+        self, uploaded_dv_class_scope, storage_class_name_scope_class
+    ):
+        dv = uploaded_dv_class_scope
+        if sc_volume_binding_mode_is_wffc(sc=storage_class_name_scope_class):
+            create_dummy_first_consumer_pod(dv=dv)
+        dv.wait_for_dv_success()
```

Comment:
Why do we need the first consumer pod? 
We create a DV, upload an image, and create a VM - this looks more like a user flow